### PR TITLE
Finished Crappy Day Message

### PR DESCRIPTION
**Pull request by memcmahon**
Some detailed review of what is in the PR. Hey @aziobrow can you review this afternoon?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Hello World
+
+Its a real crappy day out there.
```
Comment (aziobrow) [inline README.md]: Should this be "it's"? This is a contraction.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Hello World
```
Comment (aziobrow) [inline README.md:1]: I like this header-- it says exactly what it means.

**Review by aziobrow — CHANGES_REQUESTED** [review]
I requested changes for the contraction on line 3. Otherwise, great work.

File: README.md (hunk 1 above)
Comment (memcmahon) [reply README.md]: Great catch, updated to 'it's'

**Pull request merged**